docs/quick-start-guide.adoc

```diff
@@ -3,21 +3,21 @@
 +
 **On Linux:**
 
-    $ curl -L -o cm https://github.com/aerokube/cm/releases/download/1.1.2/cm_linux_amd64
+    $ curl -L -o cm https://github.com/aerokube/cm/releases/download/1.2.1/cm_linux_amd64
```

Comment:
- give the latest releases link instead of exact asset
- use variable `${LATEST_RELEASE}` instead of exact num
- use revnumber

choose what u like more :)